modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -264,7 +263,8 @@ class NDB_Form_user_accounts extends NDB_Form
         
         // email address
         $group[] = $this->createText('Email');
-        $group = array_merge($group, $this->createCheckbox('SendEmail', 'Send email to user'));
+        //$group = array_merge($group, $this->createCheckbox('SendEmail', 'Send email to user'));
```

Comment:
Thanks for your comments, I deleted the unused code that was a leftover from debugging.